rasa/core/training/interactive.py

```diff
@@ -1382,23 +1382,24 @@ def _print_help(skip_visualization: bool) -> None:
             )
             return
 
-        trackers = await training.load_data(
-            stories,
-            Domain.from_dict(domain),
-            augmentation_factor=0,
-            use_story_concatenation=False,
-        )
-
         intents = [next(iter(i)) for i in (domain.get("intents") or [])]
 
         num_messages = 0
-        sender_ids = [t.events for t in trackers] + [sender_id]
 
         if not skip_visualization:
-            plot_file = "story_graph.dot"
-            await _plot_trackers(sender_ids, plot_file, endpoint)
+            events_including_current_user_id = await _get_trackers_to_plot(
```

Comment:
Not sure about the names. `_get_trackers_to_plot` sound like it returns something with trackers, but then the variable it is stored to is called `events_including_current_user_id`.

Reply:
Mhm, its nested list of events. How about `_get_tracker_events_to_plot`?